README.md

```diff
@@ -1,2 +1,28 @@
-# terraform-aws-ses-sending-domain
-Terraform module creating an SES sending domain
+# Terraform module creating AWS SES sending domain
+
+This module configures a sending domain for SES allowing email addresses with
+that domain to be used in the `From` field of the emails sent via SES.
+
+This includes the following components:
+
+* [SES domain verification](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/verify-domain-procedure.html)
+* [Bounce and complaint notification to SNS](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/monitor-sending-activity-using-notifications-sns.html)
+* [SPF](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/send-email-authentication-spf.html)
+* [DKIM](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/send-email-authentication-dkim-easy-setup-domain.html)
+
+## Example
+
+```tf
+module "ses-sending-domain-example" {
+  source  = "babbel/ses-sending-domain/aws"
+  version = "~> 1.0"
+
+  domain_name     = "example.com"
+  route53_zone_id = "A123456789"
+  sns_topic_name  = "example"
```

Comment:
We could also use `service` here, the fact that an SNS topic is created may be an implementation detail.

Reply:
@awendt As `sns_topic_name` is used exactly once for the name of the `aws_sns_topic`, I'd recommend to keep the name.

A theoretical alternative might to be also to derive the SNS topic name from `var.domain_name`, but that's also kind of ugly because we'd have to replace the `.` with `-` or `_`.